src/components/Info.js

```diff
@@ -1,42 +1,97 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import styled from 'styled-components'
 import { breakpoint } from '../microsite-logic'
+import { FIRST_TERM, PREACTIVATION_END } from '../utils'
 import infoBackground from './assets/info-background.svg'
 import anj from './assets/anj-logo.png'
 
 const large = css => breakpoint('large', css)
 const medium = css => breakpoint('medium', css)
 
-const Info = () => (
-  <InfoSection>
-    <InfoWrapper>
-      <TextContainer>
-        <TitleWrapper>
-          <Img src={anj} alt="" />
-          <h2 className="pink header">
-            Pre-activation ends when Aragon Court Launches on Feb 10th
-          </h2>
-        </TitleWrapper>
-        <br />
-        <p className="content">
-          On February 10th, when Aragon Court's first term begins, jurors will
-          be able to deactivate and unstake ANJ using the jurors dashboard. In
-          the meantime, you can check your active balance by connecting your
-          wallet to the{' '}
-          <a className="pink" href="#get-anj">
-            conversion module.
-          </a>
-        </p>
-        <br />
-        <p className="content">
-          When the pre-activation phase ends, ANJ will be available at a
-          variable rate. Price discovery will be automated by a bonding curve
-          and it will vary based on supply and demand.
-        </p>
-      </TextContainer>
-    </InfoWrapper>
-  </InfoSection>
-)
+const Info = () => {
```

Comment:
Should we try to follow the same styling conventions than in other apps?

```suggestion
function Info() {
```

Reply:
Done in 3b7b124.